rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -62,7 +62,10 @@ As specified by RFC015, the node MUST authenticate the peer's node DID as follow
 Certain messages in the protocol are a response to a certain request. To make sure a response corresponds to a certain request, a `conversationID` is added to the request. Response type messages MUST add the same `conversationID` in the response.
 The `conversationID` is scoped to a connection.
 A node is free to choose the form of a `conversationID`, it MUST be unique during the lifetime of the connection.
-It MUST be valid for at least 10 seconds and no longer than 30 seconds.
+It MUST be valid for at least 10 seconds.
+It MUST be invalidated after 30 seconds of the last processed message.
+If a lot of messages have to be processed, it could take a while for a conversation to finish.
+After processing a message the timeout of 30 seconds will have to be reset.
```

Comment:
To prevent a timeout during a long conversation, the timeout should be reset after handling of each _valid_ message.

No need to reset the timer or keep the conversation going if one of the messages is invalid (e.g., missing prevs)